index.html

```diff
@@ -2345,9 +2377,11 @@ <h2 id="AudioParam">
               The value during the time interval \(T_0 \leq t &lt; T_1\) (where
               \(T_0\) is the time of the previous event and \(T_1\) is the
               <code>endTime</code> parameter passed into this method) will be
-              calculated as: $$ v(t) = V_0 + (V_1 - V_0) \frac{t - T_0}{T_1 -
-              T_0} $$
+              calculated as:
             </p>
+            <pre>
+              $$ v(t) = V_0 + (V_1 - V_0) \frac{t - T_0}{T_1 - T_0} $$
```

Comment:
I'd like $$ to be treated like <p>, </p>; that is, $$ on its own line, contents indented.  But you can leave it as is for this PR.  I can fix these up afterwards.